### PR TITLE
fix colima deploy part2

### PR DESCRIPTION
**Pull request by sunkickr**
## Description

seems like the login command is still not working. This update get it to work and prints the result of running the command ( in this case "login succeeded")

## 🎟 Issue(s)

- https://github.com/astronomer/astro-cli/issues/615

## 📋 Checklist

- [X] Rebased from the main (or release if patching) branch (before testing)
- [ ] Ran `make test` before taking out of draft
- [ ] Ran `make lint` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Astro-API](https://github.com/astronomer/astro/) (if necessary).
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

File: airflow/docker_image.go [hunk 1]
```diff
@@ -299,7 +299,8 @@ var cmdExec = func(cmd string, stdout, stderr io.Writer, args ...string) error {
 func useBash(authConfig *cliTypes.AuthConfig, image string) error {
 	var err error
 	if authConfig.Username != "" { // Case for cloud image push where we have both registry user & pass, for software login happens during `astro login` itself
-		err = cmdExec(EchoCmd, nil, os.Stderr, fmt.Sprintf("%q", authConfig.Password), "|", DockerCmd, "login", authConfig.ServerAddress, "-u", authConfig.Username, "--password-stdin")
+		cmd := "echo \"" + authConfig.Password + "\"" + " | docker login " + authConfig.ServerAddress + " -u " + authConfig.Username + " --password-stdin"
+		err = cmdExec("bash", os.Stdout, os.Stderr, "-c", cmd)
```
Comment (neel-astro) [inline airflow/docker_image.go:303]: will this work on a machine that does not have bash, let's say windows powershell?
what is the root cause for the original command not working with Colima?

Comment (sunkickr) [reply airflow/docker_image.go:303]: I don't think that colima works with windows powershell

Comment (sunkickr) [reply airflow/docker_image.go:303]: with the old command I was getting `Error: Cannot perform an interactive login from a non TTY device`

Comment (sunkickr) [reply airflow/docker_image.go:303]: for some reason running the echo command with bash fixes it: https://stackoverflow.com/questions/52280905/golang-exec-command-bash-command-not-working

**Review by neel-astro — APPROVED** [review]

File: airflow/docker_image.go (hunk 1 above)
Comment (neel-astro) [reply airflow/docker_image.go:303]: let's leave a comment in the code on the drawback of the new logic

**Pull request merged**